ui/document_uploader.py

```diff
@@ -800,7 +803,7 @@ def ErrMessage(message):
     msg.exec_()
 
 
-if __name__ == "__main__":
+if __name__ == '__main__':
     app = QApplication(sys.argv)
     du = DocumentUploader()
     du.showNormal()

```

Comment:
We can remove this in the production code.

Reply:
Done @pgathogo